assets/js/blocks/checkout/cart/editor.scss

```diff
@@ -0,0 +1,18 @@
+.wc-checkout__cart {}
+
+.wc-checkout__cart-table {
+	border-collapse: collapse;
+	width: 100%;
```

Comment:
These styles are overwritten by `.editor-styles-wrapper table` ruleset in my setup:

![image](https://user-images.githubusercontent.com/3616980/59026142-03925f00-8856-11e9-83c5-ef023469170c.png)

Is that expected? In that case, do we need them?

Reply:
(just for tracking what we said in person) this is needed for better display in Twenty Nineteen.